spec/support/authentication_helper.rb

```diff
@@ -0,0 +1,16 @@
+module AuthenticationHelper
+  def sign_in_as(user)
+    visit root_path
+    click_on 'Sign In'
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_on 'Log in'
+  end
+
+  def sign_out_as(_user)
```

Comment:
You are not using the `_user` variable in this method. You should get rid of it.